linkis-commons/linkis-common/src/main/java/org/apache/linkis/common/utils/SecurityUtils.java

```diff
@@ -46,25 +47,112 @@ public abstract class SecurityUtils {
 
   private static final String QUESTION_MARK = "?";
 
+  private static final String REGEX_QUESTION_MARK = "\\?";
+
+  private static final int JDBC_URL_ITEM_COUNT = 2;
+
   /** allowLoadLocalInfile,allowLoadLocalInfiled,# */
-  public static final CommonVars<String> MYSQL_SENSITIVE_PARAMS =
+  private static final CommonVars<String> MYSQL_SENSITIVE_PARAMS =
       CommonVars$.MODULE$.apply(
           "linkis.mysql.sensitive.params",
           "allowLoadLocalInfile,autoDeserialize,allowLocalInfile,allowUrlInLocalInfile,#");
 
   /**
    * "allowLoadLocalInfile=false&autoDeserialize=false&allowLocalInfile=false&allowUrlInLocalInfile=false"
    */
-  public static final CommonVars<String> MYSQL_FORCE_PARAMS =
+  private static final CommonVars<String> MYSQL_FORCE_PARAMS =
       CommonVars$.MODULE$.apply(
           "linkis.mysql.force.params",
           "allowLoadLocalInfile=false&autoDeserialize=false&allowLocalInfile=false&allowUrlInLocalInfile=false");
 
-  public static final CommonVars<String> MYSQL_STRONG_SECURITY_ENABLE =
+  private static final CommonVars<String> MYSQL_STRONG_SECURITY_ENABLE =
       CommonVars$.MODULE$.apply("linkis.mysql.strong.security.enable", "false");
 
+  private static final CommonVars<String> MYSQL_CONNECT_URL =
+      CommonVars.apply("wds.linkis.server.mdm.service.sql.url", "jdbc:mysql://%s:%s/%s");
```

Comment:
remove wds and rename linkis.security.mysql.url.regex

Reply:
if the name  `linkis.security.mysql.url.template` is better?

Reply:
> if the name `linkis.security.mysql.url.template` is better?

yes